packages/design-system/src/_forms.scss

```diff
@@ -89,3 +89,7 @@ select.form-select,
 .form-check-label {
   user-select: none;
 }
+
+input::placeholder {
+  opacity: 0.5;
```

Comment:
That seems really low - what about `0.75` or even `0.65`? Can you provide some screenshots?

Reply:
Bootstrap's default max for placeholder opacity is `0.5` (you can find it in the `$placeholder-opacity-max` variable) and the min is `0.2`. I've attached screenshots of each below. 
**0.75**
![0 75](https://github.com/user-attachments/assets/742d8665-14ed-43be-a07a-4a78da0278a7)
**0.65**
![0 65](https://github.com/user-attachments/assets/1a51d924-dcc6-4de6-874e-7f894b9359b7)
**0.5**
![0 5](https://github.com/user-attachments/assets/7dcd1290-2027-4a80-934c-80223f174cd4)

Reply:
The `$placeholder-opacity-*` variables are for the [Placeholders component](https://getbootstrap.com/docs/5.3/components/placeholders/). Regardless, I think I like either 0.75 or 0.65.

Reply:
I dug into this a little more - Bootstrap provides [default styling for placeholders](https://github.com/twbs/bootstrap/blob/2f7184e125099c01ddfc513ba5291c085b74521a/scss/forms/_form-control.scss#L75-L79). I suspect something else might be overriding the styling, or the `--bs-secondary-color` has changed drastically. The default value for the placeholder color is `var(--bs-secondary-color)` with `opacity: 1`. However, `--bs-secondary-color` includes an opacity of 75%.

Reply:
I'm fine with setting it to 0.75 since that offers enough visual representation that the field is not filled out. We may want to discuss whether to even include placeholders at all (see the top results of [this search](https://www.google.com/search?q=should+i+include+placeholders+in+inputs&oq=should+i+include+placeholders+in+inputs&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRifBdIBCDQ5MDdqMGoxqAIAsAIA&sourceid=chrome&ie=UTF-8) - they all say no)